src/shared/media-types.ts

```diff
@@ -0,0 +1,6 @@
+enum MediaTypes {
+  MOBILE = '576px',
+  TABLET = '767px',
+}
+
+export default MediaTypes;
```

Comment:
From my point of view, it would be better to create media.ts file in configs folder.

Next, I have created the same file too and it looks like that:
```javascript
enum Size {
  MOBILE = 576,
  TABLET_M = 768,
  TABLET_L = 1024,
  DESKTOP = 1920,
}

const customMediaQuery = (maxWidth: number): string => `@media (max-width: ${maxWidth}px)`;

export const media: { [key: string]: string } = {
  mobile: customMediaQuery(Size.MOBILE),
  tabletM: customMediaQuery(Size.TABLET_M),
  tabletL: customMediaQuery(Size.TABLET_L),
  desktop: customMediaQuery(Size.DESKTOP),
};
`
It gives us possibility to write:
```javascript
 ${media.mobile} {
    flex-direction: column;
}
`
instead: 
```javascript
@media (max-width: ${MediaTypes.MOBILE}) {
    padding: 22px 37px;	    padding: 22px 37px;
  }	  }
`
It'll  be shotrer, will not it?

**Which approach is better? What do you think?**

Reply:
It gives us possibility to write:

```javascript
 ${media.mobile} {
    flex-direction: column;
}
`
instead: 
```javascript
@media (max-width: ${MediaTypes.MOBILE}) {
    padding: 22px 37px;	    padding: 22px 37px;
  }	  }
`
It'll  be shotrer, will not it?

**Which approach is better? What do you think?**

Reply:
It'll be shorter, will not it?

**Which approach is better? What do you think?**

Reply:
I think it's a good point 👍 But let's make a bit flexible.

Let's call it, like customMediaQueryMax or customMediaQueryMin, for `max-width` and `min-width`. 

```
const customMediaQueryMax = (width: number): string => `@media (max-width: ${width}px)`;

const customMediaQueryMin = (width: number): string => `@media (min-width: ${width}px)`;
```

Also, in future we could have like mixed or some custom cases: `@media (min-width: ${width}px and max-width: ${width}px)` or `@media (max-width: ${width}px and min-width: ${width}px)`. 

Not need to add them now, but we should **consider** this now.

Reply:
@oknelvapi updated PR 👍